src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Area {
```

Comment:
Нам не потрібно імплементувати інтерфейси, якщо ми наслідуємося від класу, який вже його імплементує
```suggestion
public class Circle extends Figure {
```